packages/react-storage/src/components/StorageManager/ui/FileList/FileStatusMessage.tsx

```diff
@@ -17,55 +17,83 @@ export const FileStatusMessage = ({
   uploadSuccessfulText,
 }: FileStatusMessageProps): React.JSX.Element | null => {
   const icons = useIcons('storageManager');
-  switch (status) {
-    case FileStatus.UPLOADING: {
-      return (
-        <Text className={ComponentClassName.StorageManagerFileStatus}>
-          {getUploadingText(percentage)}
-        </Text>
-      );
+
+  // Generate the screen reader announcement text
+  const announcementText = () => {
```

Comment:
Will need to provide a callback function through the `FileStatusMessage` that maps from the top level `displayText` prop to allow for consumer's to override for i18n